files/private-chef-upgrades/001/022_policyfile_schema_and_permissions.rb

```diff
@@ -0,0 +1,19 @@
+define_upgrade do
+  if Partybus.config.bootstrap_server
+    must_be_data_master
+    run_sqitch("policyfile-api-tables", "@1.0.4")
```

Comment:
@stevendanna This line would be replaced by a comment saying "previous migration added the tables we want"